src/main/java/tests/OperationHandlersTest.java

```diff
@@ -0,0 +1,65 @@
+package tests;
+
+import core.basesyntax.*;
+import org.junit.jupiter.api.Test;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class OperationHandlersTest {
+    @Test
+    public void testBalanceOperation() {
+        OperationHandler handler = new BalanceOperation();
+        Map<String, Integer> storage = new HashMap<>();
+        FruitTransaction transaction = new FruitTransaction();
+        transaction.setOperation(Operation.BALANCE);
+        transaction.setFruit("banana");
+        transaction.setQuantity(100);
+        ((BalanceOperation) handler).handle(transaction, storage);
+
+        assertEquals(100, (int) storage.get("banana"));
+    }
+
+    @Test
+    public void testSupplyOperation() {
+        OperationHandler handler = new SupplyOperation();
+        Map<String, Integer> storage = new HashMap<>();
+        FruitTransaction transaction = new FruitTransaction();
+        transaction.setOperation(Operation.SUPPLY);
+        transaction.setFruit("banana");
+        transaction.setQuantity(100);
+        handler.handle(transaction, storage);
+
+        assertEquals(100, (int) storage.get("banana"));
```

Comment:
Make sure that the `OperationHandler` interface is properly used in tests. The `handle` method should be declared in the interface and implemented in `SupplyOperation` so that you can use it without casting.